pkgs/development/python-modules/ehtim/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  stdenv, lib, buildPythonPackage, fetchFromGitHub,
+  numpy, scipy, astropy, matplotlib, ephem, h5py, pandas, requests, future
+}:
+
+buildPythonPackage rec {
+  pname = "ehtim";
+  version = "1.2.4";
+
+  src = fetchFromGitHub {
+    owner = "achael";
+    repo = "eht-imaging";
+    rev = "v${version}";
+    sha256 = "ZeacOUve1QHX4Ei61GHEMtbqyfOV7Cj9q7+sH/lQR+w=";
+  };
+  patches = [ ./silence.patch ];
+
+  propagatedBuildInputs = [
+    numpy scipy astropy matplotlib ephem h5py pandas requests future
+  ];
+
+  doCheck = false;
```

Comment:
Why is the check phase disabled?

Reply:
Upstream does not have many tests and virtually all of them require additional data. I cannot even run the tests myself. Also something that needs to be discussed with upstream.

Reply:
Please add a comment saying something like `# Tests require additional data.` if that's the case.

Reply:
Done.